elasticsearch/client_test.go

```diff
@@ -144,7 +155,9 @@ func TestGetStatus(t *testing.T) {
 				},
 			}
 
-			client := New("http://localhost:999", dphttpMock, false)
+			var testSigner *esauth.Signer
+
+			client := New("http://localhost:999", dphttpMock, false, testSigner, "es", "eu-west-1")
```

Comment:
Can you also add a new test case to cover the `signRequests == true` && `signer = nil` please? 😄